pyCGM_Single/pyCGM.py

```diff
@@ -704,59 +719,42 @@ def footJointCenter(frame,vsk,ankle_JC,knee_JC,delta):
-        axis components. The xyz axis components are 2x3 arrays consisting of the axis center
+        Returns an array that contains the foot axis center in a :math:`1 \\times 3` array of xyz values,
+        which is then followed by a :math:`2 \\times 3 \\times 3` array composed of the foot axis center x, y, and z
+        axis components. The xyz axis components are :math:`2 \\times 3` arrays consisting of the axis center
         in the first dimension and the direction of the axis in the second dimension.
         This function also saves the static offset angle in a global variable.
 
-    Modifies
-    --------
-    Axis changes following to the static info.
-
-    you can set the static_info by the button. and this will calculate the offset angles
-    the first setting, the foot axis show foot uncorrected anatomical reference axis(Z_axis point to the AJC from TOE)
-
-    if press the static_info button so if static_info is not None,
-    and then the static offsets angles are applied to the reference axis.
-    the reference axis is Z axis point to HEE from TOE
-
     Examples
     --------
     >>> import numpy as np
     >>> from .pyCGM import footJointCenter
-    >>> vsk = { 'RightStaticRotOff' : 0.015683497632642047, 'LeftStaticRotOff': 0.009402910292403012,
-    ...         'RightStaticPlantFlex' : 0.2702417907002758, 'LeftStaticPlantFlex': 0.20251085737834015}
-    >>> frame = { 'RHEE': np.array([374.01257324, 181.57929993, 49.50960922]),
-    ...           'LHEE': np.array([105.30126953, 180.2130127, 47.15660858]),
-    ...           'RTOE': np.array([442.81997681, 381.62280273, 42.66047668]),
-    ...           'LTOE': np.array([39.43652725, 382.44522095, 41.78911591])}
-    >>> knee_JC = [np.array([364.17774614, 292.17051722, 515.19181496]),
-    ...           np.array([143.55478579, 279.90370346, 524.78408753]),
-    ...           np.array([[[364.64959153, 293.06758353, 515.18513093],
-    ...           [363.29019771, 292.60656648, 515.04309095],
-    ...           [364.04724541, 292.24216264, 516.18067112]],
-    ...           [[143.65611282, 280.88685896, 524.63197541],
-    ...           [142.56434499, 280.01777943, 524.86163553],
-    ...           [143.64837987, 280.04650381, 525.76940383]]])]
-    >>> ankle_JC = [np.array([393.76181608, 247.67829633, 87.73775041]),
-    ...            np.array([98.74901939, 219.46930221, 80.6306816]),
-    ...            [[np.array([394.4817575, 248.37201348, 87.715368]),
-    ...            np.array([393.07114384, 248.39110006, 87.61575574]),
-    ...            np.array([393.69314056, 247.78157916, 88.73002876])],
-    ...            [np.array([98.47494966, 220.42553803, 80.52821783]),
-    ...            np.array([97.79246671, 219.20927275, 80.76255901]),
-    ...            np.array([98.84848169, 219.60345781, 81.61663775])]]]
+    >>> vsk = { 'RightStaticRotOff' : 0.0, 'LeftStaticRotOff': 0.0,
```

Comment:
This would be better as 4 digits.  While the other values, like 'RHEE' are marker positions in mm, these offset values are in degrees and a small multiplier does have an impact.